java/arcs/sdk/wasm/WasmMultipleHandles.kt

```diff
@@ -0,0 +1,781 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.sdk.wasm
+
+/** Combined Handle to allow events on two handles to trigger actions. */
+class WasmTupleHandle<T, U>(
```

Comment:
As commented on https://github.com/PolymerLabs/arcs/pull/4835 I don't understand why we're having classes in this file.

What other things you will be able to do after combine(a, b, c), and what are the benefit of introducing these classes vs. exposing methods for attaching callbacks directly.
```
combineUpdates(Handle<A>, Handle<B>, Handle<C>) {a, b, c ->
(...)
}
```

Reply:
I like this idea!